### ISSUE TITLE
All the mods 6 1.16.4 does not like most resource packs

### ISSUE THREAD

**Issue by Marker248** [AllTheMods/ATM-6#405]
Not sure if this is specific to the mod pack or to 1.16.4 to be honest. I only just started using it this week.
I've been trying to load Minecraft with select resource packs but I keep getting a "resource reload failed" message in the top right and when I go to check what is loaded it is only the default resource packs. It is also not consistent in it's behavior, some resource packs seem to work sometimes but others completely freeze the game and I have to force close it.
I also wanted to make note of the incompatible built in pack that I have been seeing since installing the modpack on the launcher.
![image](https://user-images.githubusercontent.com/9679256/100309011-0f507480-2f67-11eb-916e-aec2f63f5e87.png)

Can that be removed? or is it a factor in the resources packs not loading?

To date I have tried Soartex, LBPR reload and Misa realistic, all with less than acceptable results. Nothing higher than x128 as well

Any hints or tips would be appreciated.

**Comment by NetMasterFlex** [comment]
What Version of ATM6 are you currently using? I had an issue getting Soartex Fanver to load when the pack was on 1.16.3.  Come to find out thru trial and error I found that there was an issue with resource packs of any kind and Immersive Engineering.  I manually updated that modpack alone from I think it was .127 to .128 and I was able to successfully apply Soartex Fanver.  ATM6 Devs updated the modpacks on the next version and I've have success ever since.  I use the Soartex Fanver 8.0.2 downloaded from curseforge.  I'm currently on ATM6 v 1.3.2c.  Optifine however is broken so if you are attempting to introduce optifine along with your resource pack that might be part of the problem.  Also the Optifine issue is not an ATM6 modpack problem and doubt will be addressed in this GitHub.  It's more of a Forge compatibility issue.
